tests/utils.py

```diff
@@ -722,7 +722,7 @@ def load_kasvs_ecdh_vectors(vector_data):
         if line.startswith("["):
             tag = line.split()[0][1:]
         elif line.startswith("COUNT = "):
-            data["COUNT"] = int(line.split("=")[1], 16)
+            data["COUNT"] = int(line.split("=")[1])
```

Comment:
You mean this: https://github.com/pyca/cryptography/blob/master/tests/test_utils.py#L3053 ?